static/app/views/issueDetails/streamline/sidebar/solutionsSection.tsx

```diff
@@ -132,7 +132,9 @@ export default function SolutionsSection({
     >
       <SolutionsSectionContainer>
         {aiConfig.needsGenAIConsent ? (
-          <Summary>{t('Explore potential root causes and solutions with Seer.')}</Summary>
+          <Summary>
+            {t('Explore potential root causes and solutions with Autofix.')}
```

Comment:
there's a chance a user sees this on an issue that would have a summary and not autofix, so they set up and then don't see autofix, so _technically_ this might be a problem, but i don't think it's important enough. we'll change this in the future anyway most likely